packages/scandipwa/src/route/UrlRewrites/UrlRewrites.component.js

```diff
@@ -53,10 +53,19 @@ export class UrlRewrites extends PureComponent {
 
     renderContent() {
         const { props, type } = this.props;
+        const {
+            history: {
+                location: {
+                    state: {
+                        pageKey = null
+                    } = {}
+                } = {}
+            } = {}
+        } = props;
 
         switch (type) {
         case TYPE_PRODUCT:
-            return <ProductPage { ...props } />;
+            return <ProductPage { ...props } key={ pageKey } />;
```

Comment:
Why `key` prop was put here? And is it okay that it can be `null`?

Reply:
The key is used here in order to rerender product page, so it would be displayed from the top.
If you click on related product on other product page, page is not rerendered. Only some parts of the page are rerendered with the new data.
It's ok that the key is null in this case, since the ProductPage component is only one and there will not be any duplicate keys.

Reply:
Okay, thanks for the clarification!